VirtualReality/MRML/vtkVirtualRealityViewInteractorObserver.cxx

```diff
@@ -219,6 +219,9 @@ bool vtkVirtualRealityViewInteractorObserver::DelegateInteractionEventDataToDisp
     ed->SetAccuratePicker(vrViewInteractorStyle->GetAccuratePicker());
     }
 
+  // Assumes event are all 3D and invalidates display position
+  ed->SetDisplayPositionInvalid();
```

Comment:
You should not have to do this. Based on our discussion, it looks like this is native to Slicer's bookkeeping of whether to compute the world position again. See https://github.com/Slicer/Slicer/blob/4ed0396cf865870dffe887e28db588434ce23b02/Libs/MRML/DisplayableManager/vtkMRMLInteractionEventData.cxx#L356C79-L358

Reply:
All events have 3D and display position, too. Display position may be less useful in virtual reality (e.g., a Google-glass-like 2D overlay), but we may use it in the future. The flags for valid "accurate" and "display" positions are indeed for avoiding (potentially costly) recomputations. The same event is given to many displayable managers, so we can save time by computing it only once and storing the result.

Reply:
> but we may use it in the future

That is sensible.

Should we simply removes this method then ? 

For reference, it was originally introduced here:

https://github.com/Slicer/Slicer/commit/a0ad2a32824dee03c15182c00d71391b11dc80c2#diff-3c145879816516bad207b32924f5bb882c68f6c0de514970fca6f372eee76b86R290-R295